src/ansys/fluent/core/post_objects/post_helper.py

```diff
@@ -131,11 +134,21 @@ def get_vector_fields(self):
 
     def get_field_unit(self, field):
         """Returns the unit of the field."""
-        quantity = self._field_unit_quantity(field)
-        if quantity == "*null*":
-            return ""
-        scheme_eval_str = f"(units/get-pretty-wb-units-from-dimension (units/inquire-dimension '{quantity}))"
-        return " ".join(self._scheme_str_to_py_list(scheme_eval_str))
+        session = self.obj.get_root().session
+        if hasattr(session, "scheme_eval"):
```

Comment:
not sure why this check is required, we are already checking the fluent version to choose between scheme_eval and field_data based implementation

Reply:
This check is present is `_scheme_str_to_py_list` therefore we have followed it. I think we can remove it from there as well.